class-8-js-scopes/ziada-deniz/index.js

```diff
@@ -0,0 +1,73 @@
+// 1- How to make the variable `result` accessible outside the function `add`? Need to declare on the global scope
+
+let result;
+function add(a, b) {
+   result = a + b;
+}
+
+add(2, 4);
```

Comment:
the result of the function is undefined, this needs to be fixed

Reply:
@Ammar-64 
Revised and done!